resolwe_bio/processes/WGBS/bsrate.py

```diff
@@ -0,0 +1,107 @@
+"""Calculate conversion rate for bisulfite sequencing."""
+import os
+
+from plumbum import TEE
+
+from resolwe.process import (
+    BooleanField,
+    Cmd,
+    DataField,
+    FileField,
+    FloatField,
+    IntegerField,
+    Process,
+    SchedulingClass,
+)
+
+
+class BsConversionRate(Process):
+    """Estimate bisulfite conversion rate in a control set.
+
+    The program bsrate included in [Methpipe]
+    (https://github.com/smithlabcode/methpipe) will estimate the bisulfite
+    conversion rate.
+    """
+
+    slug = 'bs-conversion-rate'
+    name = 'Bisulfite conversion rate'
+    process_type = 'data:wgbs:bsrate'
+    version = '1.0.0'
+    scheduling_class = SchedulingClass.BATCH
+    entity = {'type': 'sample'}
+    requirements = {
+        'expression-engine': 'jinja',
+        'executor': {
+            'docker': {
```

Comment:
Estimate the resources needed and specify the `cores` and `memory` explicitly.

Reply:
I have not found any evidence that multiple cores improve speed. 16GB sufficed in tests that I ran on google cloud.